docs/modules/ROOT/pages/release-notes/release-notes-2020-7.adoc

```diff
@@ -0,0 +1,79 @@
+Edition, we have removed the production domain from the distributions.
+The template for this domain
+(`common/templates/gf/production-domain.jar`) is still available,
+although this too will be removed in a future release. Note that Payara
+Enterprise Edition will still contain the production domain.
+
+== New Features
+
+* [https://github.com/payara/Payara/pull/4970[FISH-330]] - Implement
+LDAP MicroProfile Config Source
+* [https://github.com/payara/Payara/pull/4982[FISH-338]] - Create
+Hashicorp Cloud Vault MicroProfile Config Source
+* [https://github.com/payara/Payara/pull/4986[FISH-660]] - MP Health 3.0
+Upgrade
+* [https://github.com/payara/Payara/pull/4988[FISH-661]] - MP JWT Auth
+1.2 Upgrade
+
+== Improvements
+
+* [https://github.com/payara/Payara/pull/4966[FISH-327]] - Implement
+MicroProfile Sniffer and Engine
+* [https://github.com/payara/Payara/pull/4990[FISH-375]] - Remove
+production domain from community version
+* [https://github.com/payara/Payara/pull/5008[FISH-651]] - Allow JWT
+verification to skip type validation
+* [https://github.com/payara/Payara/pull/4991[FISH-732]] - Improve
+message when same name is used in metrics.xml for exposed AMX bean.
+
+== Security fixes
+
+* [https://github.com/payara/Payara/pull/4977[FISH-731]] - Upgrade
+Hibernate Validator to 6.1.5.Final
+
+== Bug Fixes
+
+* [https://github.com/payara/Payara/pull/4981[FISH-206]] - @RolesAllowed
+annotation in method prevents unauthenticated calls to other methods in
+remote EJB
+* [https://github.com/payara/Payara/pull/5015[FISH-275]] - Deployment
+fails if system properties in persistence.xml are only defined in the
+target config and not in DAS config
+* [https://github.com/payara/Payara/pull/4966[FISH-431]] - OpenAPI
+document shows ejb-timer-service-app as server URL when EJB timer
+available
+* [https://github.com/payara/Payara/pull/4974[FISH-657]] - no
+<deploymentgroup> element When deploy WebServices
+* [https://github.com/payara/Payara/pull/4993[FISH-747]] - Azure Secret
+Secret Configuration screen on Web Admin console fails
+* [https://github.com/payara/Payara/pull/4440[FISH-761]] - [Community -
+https://github.com/bjetal[bjetal]] Deployment of unpacked WAB fails when
+using Felix fileinstall
+* [https://github.com/payara/Payara/pull/4933[FISH-763]] - [Community -
+https://github.com/sgflt[sgflt]] Embedded Payara is unable to start
+* [https://github.com/payara/Payara/pull/4479[FISH-766]] - [Community -
+https://github.com/sgflt[sgflt]] Improper synchronization of session map
+* [https://github.com/payara/Payara/pull/4998[FISH-767]] - Missing
+ejb-opentracing.jar in manifest of gf-client.jar
+* [https://github.com/payara/Payara/pull/4923[FISH-777]] - [Community -
+https://github.com/avpinchuk[avpinchuk]] Exclude OpenAPI rest endpoint
+from tracing
+* [https://github.com/payara/Payara/pull/4610[FISH-779]] - [Community -
+https://github.com/borisheithecker[borisheithecker]] Honor non-standard
+realm name for client-cert login
```

Comment:
Need to remove this one from the release notes pending new PR in community for Jakarta EE fix